core/src/main/java/com/google/googlejavaformat/java/Main.java

```diff
@@ -207,6 +207,7 @@ private int formatFiles(ArgInfo argInfo, JavaFormatterOptions options) {
     int numThreads = Math.min(MAX_THREADS, argInfo.parameters.fileNamesFlag.size());
     ExecutorService executorService = Executors.newFixedThreadPool(numThreads);
 
+    Map<Path, String> inputs = new LinkedHashMap<>();
```

Comment:
Instead of creating a second map, what do you think about changing `FormatFileCallable` to return a result that includes information about whether the file changed as well as the formatted output?